po/ja.po

```diff
@@ -1464,73 +1462,73 @@ msgstr "グリッドを表示する/隠す"
 #: js/turtledefs.js:144
 #: js/turtledefs.js:189
 msgid "Settings"
-msgstr "設定"
+msgstr "せってい"
 
 #: js/turtledefs.js:144
 #: js/turtledefs.js:189
 msgid "Open a panel for configuring Music Blocks."
-msgstr "ミュージック・ブロックスを設定するためのパネルを開きます。"
+msgstr "ミュージック・ブロックスの設定メニューを開く。"
 
 #: js/turtledefs.js:145
 #: js/turtledefs.js:197
 msgid "Restore blocks from the trash."
-msgstr "ブロックをゴミ箱から取り戻します。"
+msgstr "ゴミ箱にすててしまったブロックを取り出してもどす。ふくすうのブロックをすててあるときは新しい順に、ゴミ箱が空になるまでブロックを拾いもどすことができる。"
 
 #: js/turtledefs.js:146
 #: js/turtledefs.js:190
 msgid "Switch mode"
-msgstr "モードを設定"
+msgstr "はってんモード／かんたんモードにする"
 
 #: js/turtledefs.js:146
 #: js/turtledefs.js:190
 msgid "Switch between beginner and advance modes."
-msgstr "簡単/高度モードを設定"
+msgstr "ミュージック・ブロックスをかんたんモード／はってんモードに切りかえる。モードによって使えるきのうやブロックの種類が変わる。"
 
 #: js/turtledefs.js:147
 #: js/turtledefs.js:196
 msgid "Select your language preference."
-msgstr "言語を選んでください"
+msgstr "ブロックの名前などに表示される言語を選ぶ。"
 
 #: js/turtledefs.js:148
 #: js/turtledefs.js:192
 msgid "Decrease the size of the blocks."
-msgstr "ブロックの大きさを小さくします。"
+msgstr "画面に表示されるブロックのサイズを小さくする。"
 
 #: js/turtledefs.js:149
 #: js/turtledefs.js:193
 msgid "Increase the size of the blocks."
-msgstr "ブロックの大きを大きくします。"
+msgstr "画面に表示されるブロックのサイズを大きくする。"
 
 #: js/turtledefs.js:150
 #: js/turtledefs.js:174
 msgid "Clear the screen and return the mice to their initial positions."
-msgstr "画面をクリアしてネズミをみんな元に戻す"
+msgstr "ネズミを元の位置にもどし、ペンでえがいた線をすべて消す。"
 
 #: js/turtledefs.js:151
 #: js/turtledefs.js:175
 msgid "Collapse the graphics window."
-msgstr "グラフィックスウィンドーを非表示"
+msgstr "ネズミがいどうしたり、ペンで線をえがいたりできる「カンバス」の表示サイズを　しゅくしょう／かくだいする。<br>カンバスをしゅくしょうした場合は、プログラムをふつうのそくどで実行しても、ブロックがかくれない。ふつうの実行そくどでプログラムの動作かくにんをしたいときなどに便利だ。"
 
 #: js/turtledefs.js:152
 #: js/turtledefs.js:198
 msgid "Keyboard shortcuts"
-msgstr "ショートカット・キー"
+msgstr "キーボードのショートカット"
 
 #: js/turtledefs.js:152
 #: js/turtledefs.js:198
 msgid "You can type \"d\" to create a \"do\" block, \"r\" to create a \"re\" block, etc."
-msgstr "「d」を打つと「ド」のブロックが作られ、「r」を打つと「レ」のブロックが作られます"
+msgstr "キーボードを使うと、パレットボタンからブロックをドラッグして配置するだけでなく、ボタンをおすだけでちょくせつブロックを置くことができる。<br><br>★ショートカットキー<br>d ……　「ド」（４分音符、４オクターヴ）<br>r ……　「レ」（４分音符、４オクターヴ）<br>m ……　「ミ」（４分音符、４オクターヴ）<br>f ……　「ファ」（４分音符、４オクターヴ）<br>s ……　「ソ」（４分音符、４オクターヴ）<br>l ……　「ラ」（４分音符、４オクターヴ）<br>t ……　「シ」（４分音符、４オクターヴ）<br>"
 
 #: js/turtledefs.js:153
 #: js/turtledefs.js:199
 msgid "Guide"
-msgstr "ガイド"
+msgstr "もっとくわしく知るには"
 
 #: js/turtledefs.js:153
 #: js/turtledefs.js:199
 ## しょうさい＝詳細
 msgid "A detailed guide to Music Blocks is available."
-msgstr "ミュージック・ブロックスの詳細ガイドがあります。"
+msgstr "インターネットから、ミュージック・ブロックスのさらにくわしいガイドページを見ることができる。次のリンク先を開いてみよう。<br><a href="https://github.com/sugarlabs/musicblocks/tree/master/guide-ja">https://github.com/sugarlabs/musicblocks/tree/master/guide-ja</a>"
```

Comment:
@walterbender is this okay?

Reply:
Is the syntax

```
<br><a href="https://github.com/sugarlabs/musicblocks/tree/master/guide-ja">https://github.com/sugarlabs/musicblocks/tree/master/guide-ja</a>
```
within this .po file okay or will it break things?

Reply:
It is OK. You may want to add target="_blank" to the <a > tag.

Reply:
Oh. And you will need to \" the quote marks.